ITE/5/README.adoc

```diff
@@ -0,0 +1,182 @@
+of both solutions to easily integrate them.
+
+However, as time has shown, keeping these two separately-specified signature
+wrappers (i.e., one for TUF and one for in-toto) is rather difficult. As time
+passes, and due to the interactions in each community, the signature wrappers
+evolve to better fit their user-bases. Adopting a common source of truth (i.e.,
+a separate signature specification) should help increase cohesion between
+projects while maintaining the original goal of code-reuse and transparent
+integration.
+
+In addition, keeping the signature wrapper specification *outside* of the
+current in-toto specification will also simplify the specification, which can
+now only focus on describing in-toto specifics, rather than cryptographic
+building blocks.
+
+[[reasoning]]
+== Reasoning
+
+Our goal was to adopt a signature wrapper that is as simple and foolproof as
+possible. Alternatives such as [JWS] are extremely complex and error-prone,
+while others such as [PASETO] are overly specific. (Both are also
+JSON-specific.) We believe the SSL signing spec strikes the right balance of
+simplicity, usefulness, and security. 
+
+Further, the SSL signing spec is a "natural evolution" of the current signature
+wrapper, which was both defined in the TUF and in-toto specifications. As such,
+it allows for a transparent upgrade via their cryptographic provider,
+link:https://github.com/secure-systems-lab/securesystemslib[securesystemslib].
+
+Finally, since this is an improved version of the current TUF and in-toto
+wrappers, the design allows for a backwards-compatible JSON representation,
+which would allow for providers/verifiers to continue using metadata with the
+old signature wrapper. This way, an upgrade path for existing implementations
+exists without ad-hoc measures.
+
+Further information on the reasoning behind the wrapper's specifics is provided in the link:https://github.com/secure-systems-lab/signing-spec#reasoning[signing specification] repository
+
+[[Security]]
+== Security
+
+At first sight this proposal is central to security, yet the actual
+contribution is to allow for a signature provider to be disaggregated from the
+specification. As such, no supply-chain security properties are removed from
+the system through this ITE.
+
+The specifics on the signature wrapper provide, at worst, the same security
+properties through its backwards-compatible json representation. At best, the
+changes improve the security stance of implementations since it is not parsing
+untrusted input anymore.
+
+[[infrastructure-requirements]]
+== Infrastructure Requirements
+
+Since this is a change in the metadata format, no infrastructure requirements
+exist.
+
+
+[[prototype-implementation]]
+== Prototype Implementation
+
+See link:https://github.com/secure-systems-lab/signing-spec/blob/master/reference_implementation.ipynb[reference_implementation.ipynb] for an implementation.
+
+[[Testing]]
+== Testing
+
+=== Test Vector
```

Comment:
Would it make more sense to only have this test vector in the SSL signing spec and then _not_ reference it here?

(I'm think this isn't really in-toto specific at this point so it can be delegated to the signing spec.)

Reply:
Hmm, fair point. Let me move this around.